src/utils/screenshotsaver.cpp

```diff
@@ -60,16 +60,13 @@ bool ScreenshotSaver::saveToFilesystemGUI(const QPixmap &capture) {
         QString savePath = QFileDialog::getSaveFileName(
                     nullptr,
                     QString(),
-                    FileNameHandler().absoluteSavePath() + ".png");
+                    FileNameHandler().absoluteSavePath() + ".png",
+					QLatin1String("Portable Network Graphic file (PNG) (*.png);;BMP file (*.bmp);;JPEG file (*.jpg)"));
 
         if (savePath.isNull()) {
             break;
         }
 
-        if (!savePath.endsWith(QLatin1String(".png"))) {
```

Comment:
Wouldn't be better to verify if the file has a valid extension instead of removing that check?

That piece of code was added to address #278 this would introduce a regression.

Reply:
Well... I have tested my code in Windows, and it's works fine even in described case (extension adding automatically to string) and I thought shoud be the same behavior for ahother OSs but its not true indeed. I have added mentioned verification for all supported images format.